assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/table/HeaderCell.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { parseISO } from 'date-fns';
+import classNames from 'classnames';
+
+import { getBackgroundColorClassName } from '@sharedEntities/tickets/helpers';
+import { RenderCellProps } from '../../types';
+import { useMoneyDisplay } from '@appServices/utilities/money';
+import { useTimeZoneTime } from '@appServices/hooks';
+
+const HeaderCell: React.FC<RenderCellProps> = ({ ticket }) => {
+	const bgClassName = getBackgroundColorClassName(ticket);
+	const { currency } = useMoneyDisplay();
+	const { formatForSite: format } = useTimeZoneTime();
+	const startDate = format(parseISO(ticket.startDate), 'MMM dd yyyy');
+	const startDateClassName = classNames(bgClassName, 'header-cell-after-content');
```

Comment:
there was an overriding happening: 
![image](https://user-images.githubusercontent.com/1477453/78688141-9def8300-78fd-11ea-9ba5-5d7b4389466d.png)

Reply:
and I ended up not using `.ee-focus-priority-6`, but instead copied needed properties and applied to needed element:

![image](https://user-images.githubusercontent.com/1477453/78688277-c4152300-78fd-11ea-9e2b-70e1a2c10c2e.png)

Reply:
ya that override is because I don't actually want that blackish text on the green background.
the way that the auto a11y colour generation code works is that it uses the first a11y acceptable colour it finds while working away from the background color. So in this case, that green background is just light enough that it results in that black text, but the white is also a11y friendly and matches all of the other status colour schemes